packages/wallets/src/components/WalletHeaderActions/WalletHeaderActions.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import './WalletHeaderActions.scss';
+import IcCashierAdd from '../../public/images/ic-cashier-deposit.svg';
+import IcCashierWithdrawal from '../../public/images/ic-cashier-withdrawal.svg';
+import IcCashierTransfer from '../../public/images/ic-cashier-transfer.svg';
+import IcCashierStatement from '../../public/images/ic-cashier-statement.svg';
+import { useWalletAccountsList } from '@deriv/api';
+
+const getWalletHeaderButtons = (is_demo: boolean, handleAction?: () => void) => {
+    const buttons = [
+        {
+            name: 'Deposit',
+            text: is_demo ? 'Reset balance' : 'Deposit',
+            icon: <IcCashierAdd />,
+        },
+        {
+            name: 'Withdraw',
+            text: 'Withdraw',
+            icon: <IcCashierWithdrawal />,
+        },
+        {
+            name: 'Transfer',
+            text: 'Transfer',
+            icon: <IcCashierTransfer />,
+        },
+        {
+            name: 'Transactions',
+            text: 'Transactions',
+            icon: <IcCashierStatement />,
+        },
+    ];
+
+    // Filter out the "Withdraw" button when is_demo is true
+    const filteredButtons = is_demo ? buttons.filter(button => button.name !== 'Withdraw') : buttons;
+
+    return filteredButtons.map(button => ({
+        ...button,
+        action: () => handleAction?.(),
+    }));
+};
+
+const WalletHeaderActions = ({
```

Comment:
Nit: Can we rename it to more generic name `WalletActions`, cause we will use it in responsive also